onnxscript/function_libs/torch_lib/ops/special.py

```diff
@@ -219,7 +217,7 @@ def aten_special_log_ndtr(self: TensorType) -> TensorType:
 def aten_special_log_softmax(self: TFloat, dim: int, dtype: int = -1) -> TFloat:
     """special_log_softmax(Tensor self, int dim, *, ScalarType? dtype=None) -> Tensor"""
 
-    self_is_scalar = IsScalar(self)
+    self_is_scalar = len(self.shape) == 0
```

Comment:
Does removing IsScalar potentially make the output axes static?

Reply:
It does not. This is because the rank of a tensor in pytorch is always fixed.